docs/source/commandline/commandline-all.rst

```diff
@@ -1097,19 +1200,19 @@ Core Input Options
 
 **\-natural**
 
-        Allows user to specify whether or not to use a natural keyboard or not. This allows you to start your game or system in a 'native' mode, depending on your region, allowing compatability for non-"QWERTY" style keyboards. The default is OFF (*-nonatural*)
+        Allows user to specify whether or not to use a natural keyboard or not. This allows you to start your  system in a 'native' mode, depending on your region, allowing compatability for non-"QWERTY" style keyboards. The default is OFF (*-nonatural*)
```

Comment:
This needs a proper explanation.

In "emulated keyboard" mode, MAME translates pressing/releasing host keys/buttons to emulated keystrokes.  When you press/release a key/button mapped to an emulated key, MAME presses/releases the emulated key.

In "natural keyboard" mode, MAME attempts to translate characters to keystrokes.  The OS translates keystrokes to characters (like it does when you type into a text editor), and MAME attempts to translate these characters to emulated keystrokes.

There are a number of unavoidable limitations in "natural keyboard" mode:
* The emulated system driver or keyboard device or  has to support it
* The selected keyboard must match the keyboard layout selected in the emulated OS
* Keystrokes that don't produce characters can't be translated (e.g. pressing a modifier on its own)
* Holding a key until the character repeats will cause the emulated key to be pressed repeatedly
* Dead key sequences are cumbersome to use at best
* It won't work at all if IME edit is involved (e.g. for Chinese/Japanese/Korean)

Reply:
Fixed on my side.